run_pylint.sh

```diff
@@ -34,6 +34,7 @@ INIT_HOOK="
 AP = os.environ.get('AUTOTEST_PATH', '/usr/local/autotest')
 sys.path.append(os.path.abspath(AP + '/..'))
 sys.path.append(os.path.abspath('.'))
+sys.path.append(os.path.abspath('dockertest'))
```

Comment:
Hrmmmm, I don't think this should be necessary.  It's assumed that ``run_pylint.sh`` is always run with CWD of repo. root.  Since the ``dockertest`` directory has a ``__init__.py`` python should recognise it as a package automaticly.  Perhaps you need to ``import output`` from w/in ``dockertest/__init__.py``?